### PR TITLE
added sessions and admin section

### PR DESCRIPTION
**Pull request by purplena**

### CODE REVIEW

File: app/Http/Controllers/GalleryController.php [hunk 1]
```diff
@@ -10,4 +12,20 @@ public function index()
     {
         return view('gallery');
     }
+
+    public function store()
```
Comment (stephdotnet) [inline app/Http/Controllers/GalleryController.php:16]: Arr you sure this belongs here? We are in gallery controller 🤔

File: app/Models/User.php [hunk 2]
```diff
@@ -17,11 +17,7 @@ class User extends Authenticatable
      *
      * @var array<int, string>
      */
-    protected $fillable = [
-        'name',
-        'email',
-        'password',
-    ];
+    protected $guarded = [];
```
Comment (stephdotnet) [inline app/Models/User.php:20]: Ahhah why not 😁 although it's a bit risky on User 🙂

File: public/images/project1.pngZone.Identifier
Comment (stephdotnet) [inline public/images/project1.pngZone.Identifier:1]: The filename is weird. Any idea why it has zone.identifier in it's name?

**Review by stephdotnet — CHANGES_REQUESTED** [review]
Some questions here but overall it's good!!

File: app/Http/Controllers/GalleryController.php (hunk 1 above)
Comment (purplena) [reply app/Http/Controllers/GalleryController.php:16]: deleted!

File: app/Models/User.php (hunk 2 above)
Comment (purplena) [reply app/Models/User.php:20]: As far as there is no registration form available and no mass assignement possible, it is okay to leave it this way

File: public/images/project1.pngZone.Identifier
Comment (purplena) [reply public/images/project1.pngZone.Identifier:1]: could not figure it out on my own.

**Pull request merged**